### PR TITLE
1657394: 2nd batch of 20 new SEO descriptions.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
## Summary

For [User Story 1657394: SEO OKR: [ms.prod(dotnet-framework)] Missing description issue (513)](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657394).

### CODE REVIEW

File: docs/framework/additional-apis/adodb.eventreasonenum.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: EventReason Enumeration (adodb)
+description: Get information about the EventReason enumeration in .NET. This interface is in the adodb namespace and the adodb assembly (in the adodb.dll library).
```
Comment (tdykstra) [inline docs/framework/additional-apis/adodb.eventreasonenum.md:3]: > the ... enumeration ... This interface

Is it an enum or an interface?

File: docs/framework/additional-apis/adodb.eventstatusenum.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: EventStatus Enumeration (adodb)
+description: Get information about the EventStatus enumeration in .NET. This interface is in the adodb namespace and the adodb assembly (in the adodb.dll library).
```
Comment (tdykstra) [inline docs/framework/additional-apis/adodb.eventstatusenum.md:3]: enum vs. interface

File: docs/framework/additional-apis/connectiongroup.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ConnectionGroup Class"
+description: Read about the ConnectionGroup class, which groups connections within the ServicePoint context & is used to maintain context for network resources in .NET.
```
Comment (tdykstra) [inline docs/framework/additional-apis/connectiongroup.md:3]:
```suggestion
description: Read about the ConnectionGroup class, which groups connections within the ServicePoint context and is used to maintain context for network resources in .NET.
```

**Review by tdykstra — APPROVED** [review]
LGTM aside from a couple minor issues noted.

File: docs/framework/additional-apis/adodb.eventreasonenum.md (hunk 1 above)
Comment (DennisLee-DennisLee) [reply docs/framework/additional-apis/adodb.eventreasonenum.md]: Yes, that was a copy/paste error. Good catch.

**Comment by DennisLee-DennisLee** [comment]
#sign-off

**Pull request merged**